src/taco-remote-lib/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "taco-remote-lib",
     "description": "Cordova-specific implementation of taco-remote's build functionality",
-    "version": "1.1.0",
+    "version": "1.1.0-dev",
```

Comment:
thanks for pointing this out.. I didn't account for tslint change which affected all the packages.